packages/hash/api/src/graphql/resolvers/knowledge/page/page.ts

```diff
@@ -72,11 +72,11 @@ export const pages: ResolverFn<
 > = async (_, { ownedById }, { dataSources: { graphApi }, userModel }) => {
   const accountModel = ownedById
     ? await UserModel.getUserById(graphApi, {
-        entityId: userAccountIdToEntityId(ownedById),
+        entityId: `${systemOrgAccountId}%${ownedById}`,
```

Comment:
We should be using this instead: https://github.com/hashintel/hash/blob/88e84beca68a835394f270350bb6d9e59e7aa62b/packages/hash/subgraph/src/types/identifier.ts#L14

Reply:
Or perhaps even define a custom function that uses the helper under the hood in the `system-org.ts` file to create an `entityId` under that account space without needing to pass `systemOrgAccountId`

Reply:
This was addressed in https://github.com/hashintel/hash/pull/1596/commits/6e6c358e80915bc8fb3fb132d66f3382fbfaf395